typescript/infra/src/config/gas-oracle.ts

```diff
@@ -0,0 +1,81 @@
+import { BigNumber, ethers } from 'ethers';
```

Comment:
I feel like some of this should live in SDK to make this accessible to PI deployers?